gulpfile.js

```diff
@@ -65,16 +68,26 @@ gulp.task('buildHighlight', function (done) {
 })
 
 gulp.task('clean', function () {
-  return del(distPath)
+  return del(distPath.html)
 })
 
-gulp.task('javascript', ['buildHighlight'], function () {
-  return gulp.src('assets/js/app.js')
+var javascript = function (options, cb) {
+  gulp.src(options.src)
     .pipe(webpack(isProduction ? config.production : config.development))
     .pipe($.if(isProduction, $.uglify()))
     .pipe($.if(isProduction, $.rev()))
     .pipe($.size({title: 'js'}))
-    .pipe(gulp.dest(distPath))
+    .pipe(gulp.dest(options.dest))
+
+  cb()
+}
+
+gulp.task('javascript', ['buildHighlight'], function () {
+  return javascript({src: 'assets/js/app.js', dest: distPath.html}, function () {});
+})
+
+gulp.task('epub-javascript', ['buildHighlight'], function () {
+  return javascript({src: 'assets/js/epub.js', dest: distPath.epub}, function () {});
 })
```

Comment:
It is fine to affect the current behaviour though since this is an internal tool.